frontend/src/pages/projects/components/PVSizeField.tsx

```diff
@@ -8,41 +8,42 @@ type PVSizeFieldProps = {
   fieldID: string;
   size: number;
   setSize: (size: number) => void;
-  disable?: boolean;
+  currentSize?: string;
 };
 
-const PVSizeField: React.FC<PVSizeFieldProps> = ({ fieldID, size, setSize, disable }) => {
-  const MIN_SIZE = 1;
+const PVSizeField: React.FC<PVSizeFieldProps> = ({ fieldID, size, setSize, currentSize }) => {
+  const minSize = parseInt(currentSize || 'NaN') || 1;
```

Comment:
huh? Why are you parsing NaN?

Reply:
https://github.com/opendatahub-io/odh-dashboard/pull/842#discussion_r1043845940 should I just simply parse `''`?